public/js/utils/source.js

```diff
@@ -29,6 +29,11 @@ function isJavaScript(url, contentType = "") {
          contentType.includes("javascript");
 }
 
+function isGenerated(source) {
```

Comment:
This check seems wrong. A "generated" source is considered any source that is the real source in the engine. This seems more like `hasSourcemap`.